src/librustc_mir/interpret/validity.rs

```diff
@@ -469,6 +480,13 @@ impl<'rt, 'a, 'mir, 'tcx, M: Machine<'a, 'mir, 'tcx>>
                         ).is_ok() ||
                         self.ecx.memory.get_fn(ptr).is_ok();
```

Comment:
Is this still needed? `check_align` looks like it handles function pointers correctly?